sql/create.go

```diff
@@ -142,6 +142,8 @@ func (p *planner) CreateIndex(n *parser.CreateIndex) (planNode, error) {
 		return nil, err
 	}
 
+	// Mark transaction as operating on the system DB.
+	p.txn.SetSystemDBTrigger()
```

Comment:
Ok. Then I think the time has come for storing more than just the transaction proto in the SQL session state. I feel strongly that this flag has no place living on the proto - it's really client state, and even more so, client state which is only used for sql.